dotnet.cmd

```diff
@@ -1,38 +1,20 @@
 @if not defined _echo @echo off
 setlocal
 
-set "__ProjectDir=%~dp0"
-set "__RepoRootDir=%__ProjectDir%..\..\"
-
 :: Clear the 'Platform' env variable for this session, as it's a per-project setting within the build, and
 :: misleading value (such as 'MCD' in HP PCs) may lead to build breakage (issue: #69).
 set Platform=
 
 :: Don't resolve runtime, shared framework, or SDK from other locations to ensure build determinism
 set DOTNET_MULTILEVEL_LOOKUP=0
 
-:: Disable first run since we do not need all ASP.NET packages restored.
+:: Disable first run since we want to control all package sources
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-set PS_DOTNET_INSTALL_SCRIPT=". %__RepoRootDir%eng\common\tools.ps1; InitializeDotNetCli($true)"
+set PS_DOTNET_INSTALL_SCRIPT=". %~dp0eng\common\tools.ps1; InitializeDotNetCli($true)"
 set "PS_COMMAND=powershell -NoProfile -ExecutionPolicy unrestricted -Command %PS_DOTNET_INSTALL_SCRIPT%"
 
-echo running: %PS_COMMAND%
 for /f "delims=" %%l in ('%PS_COMMAND%') do set "__dotnetDir=%%l"
 
-if NOT [%ERRORLEVEL%] == [0] (
```

Comment:
Is it a good idea to delete this error handling? If the SDK fails to install e.g. due the network error, is average person going to be able to figure that there was a network error?

Reply:
The arcade scripts handle such errors and print in failure cases.

Reply:
> The arcade scripts handle such errors and print in failure cases.

I do not see any errors printed by arcade scripts. Here is what I see with disconnected network:

```
C:\runtime>git clean -x -d -f .
Removing .dotnet/
Removing artifacts/

C:\runtime>dotnet
'" \dotnet.exe"' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
Yes, that's because @echo off turns off all logging which is incorrect. Thanks for pointing that out.

Reply:
Submitted a fix.